microsite/sidebars.js

```diff
@@ -541,10 +543,13 @@ module.exports = {
       },
       'api/deprecations',
       'references/glossary',
+      'overview/threat-model',
```

Comment:
We don't have one, but I was wondering if we should add a `Security` side bar item for things like these? WDYT? I would think this and maybe the audits could sit in there somewhere... 🤔

Reply:
I don't think top-level but under references would make sense 👍